src/components/QuickDropIcon.tsx

```diff
@@ -97,7 +98,7 @@ const QuickDropIcon = ({
           size={fontSize * 1.5}
           fill={isHovering ? (dark ? 'lightblue' : 'royalblue') : dark ? 'white' : 'black'}
           // disable default .icon transition so that highlight is immediate
-          style={{ cursor: 'move', transition: 'none', verticalAlign: 'middle' }}
+          cssRaw={css.raw({ cursor: 'move', transition: 'none', verticalAlign: 'middle' })}
```

Comment:
Could you please explain why `className={css({...})}` is used in most cases, but some such as this one use `cssRaw={css.raw({ ... })}`?

Pardon my ignorance on this one. I found this bit of documentation, but I'm not sure why `css.raw` applies here: https://panda-css.com/docs/guides/dynamic-styling#alternative

Reply:
This is because `css.raw({ ... })` can be passed into className like so: `className={css(cssRaw)}`. This is sometimes better than passing in className directly, since `className={cx(css({...}), className)}` doesn't merge in a reliable way (ex. if you have conflicting styles, you don't know whether the conflicting style in `className` or `css({...})` will apply, but if you do `className={cx(css({...}, cssRaw))}` instead, you know that the styles in `cssRaw` will override the `{...}` part.

Reply:
Here are some docs on this: https://panda-css.com/docs/concepts/merging-styles#merging-css-objects